go-controller/pkg/ovn/service.go

```diff
@@ -512,3 +512,27 @@ func svcQualifiesForReject(service *kapi.Service) bool {
 	_, ok := service.Annotations[OvnServiceIdledAt]
 	return !(config.Kubernetes.OVNEmptyLbEvents && ok)
 }
+
+// SVC can be of types 1. clusterIP, 2. NodePort, 3. LoadBalancer,
+// or 4.ExternalName In this scenario we will only address 1.
+// TODO adjust for upstream patch when it lands:
+// https://bugzilla.redhat.com/show_bug.cgi?id=1908540
+func getSvcVips(service *kapi.Service) []net.IP {
```

Comment:
why return an slice if you only check one ClusterIP?

Reply:
Good question... In the future I was thinking we might need to return more, but with the OVN fix I don't think we'll have to

Reply:
So actually the way I did it like this was because `gp.peerAddressSet.AddIPs()` takes a slice, so it was easier just to return it as such, and it's the precedence used by the function `GetAllPodIPs(pod)` from the util package